assets/css/custom.css

```diff
@@ -69,21 +80,82 @@ body, #qcSiteNav h4, #qcPriNav ul > li > a, #qcPriNav ul ul > li > a, .dash_titl
 
 /* ASSIGN CUSTOM PAGE BACKGROUND COLOR/IMAGE */
 
-body.home.static {
+.page-intro {
 	background: url('../img/WSC-bg.jpg') no-repeat center center fixed;
```

Comment:
thank you for this name, my calling it the "splash page" for lack of better term was just awkward 😂